Doc/library/pathlib.rst

```diff
@@ -574,6 +574,8 @@ Pure paths provide the following methods and properties:
       >>> PureWindowsPath('b.py').match('*.PY')
       True
 
+   For an overview of available patterns see :meth:`Path.glob`, with the
+   exception that "``**``" behaves just like "``*``".
```

Comment:
I think this needs to be in the first paragraph of the method description. Something like:

> Match this path against the provided glob-style pattern.  The *pattern* uses he same format as in `Path.glob()`, except that "``**``" wildcards behaves just like "``*``". Return `True` if matching is successful, `False` otherwise.